### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by roguemacaroon**
This is the home page for the ITLCT redesign project.

### CODE REVIEW

**Comment by roguemacaroon** [comment]
Land acknowledgement Z index needs to be fixed

File: css/home.css [hunk 1]
```diff
@@ -34,17 +234,640 @@ Feel free to change the min/max-width values to whatever you need. Use variables
...
+		width: 30%;
+	}
+	/* overriding global styles */
+	.events-and-programs h3,
+	.events-and-programs p {
+		color: var(--color-white);
+	}
+
+	.events-and-programs h4 {
+		color: var(--color-white);
```
Comment (maxharrisnet) [inline css/home.css:787]: The color needs to change to white on hover.

Comment (maxharrisnet) [inline css/home.css:788]: The color needs to change to white on hover.

Screen Shot 2024-01-22 at 11 57 26 AM

File: index.html [hunk 2]
```diff
@@ -1,202 +1,503 @@
...
+			<div class="partnerships-paragraph">
+				<h4>Community Partnerships</h4>
+
+				<p>
+					We partner with communities and create aligned programming and
+					events that amplify the voices, experiences, and excellence of
+					native and indigenous populations throughout the world.
+				</p>
+
+				<button class="button1">Learn More</button>
```
Comment (maxharrisnet) [inline index.html]: This button is breaking on tablet. I think it has too much padding, I have been seeing this in a few places. Feel free to edit the global style if you need to.

Screen Shot 2024-01-22 at 11 48 15 AM

File: index.html [hunk 3]
```diff
@@ -1,202 +1,503 @@
...
+		<!-- this image is only displayed on mobile and does not use a source set like the other sections -->
+		<img
+			src="./assets/images/pages/home/mobile/hero-home-mobile-2.jpg"
+			alt="a little girl holding a flower" />
+
+		<!-- "hero-home" holds the heading, paragraph and buttons for the hero image section -->
+		<!-- global class of "wrapper" is applied -->
+		<!-- "align-text" is a home-page class to center the text -->
+		<div class="hero-home wrapper align-text">
+			<h3>Preserving Tomorrow by Building Today</h3>
```
Comment (maxharrisnet) [inline index.html]: Can you please add the two buttons here? You can use # for the href value if you don't have the urls. 

Screen Shot 2024-01-22 at 12 05 01 PM

File: index.html [hunk 4]
```diff
@@ -1,202 +1,503 @@
...
+
+<body>
+	<!-- header section -->
+
+	<!-- =======================
+		Hero Section
+		========================== -->
+
+	<!-- "hero" is the class for the entire section -->
+	<section class="hero">
```
Comment (maxharrisnet) [inline index.html]: Can we add the navigation here?

**Review by maxharrisnet — CHANGES_REQUESTED** [review]
Looks great, just a few small changes to make.

File: index.html (hunk 3 above)
Comment (roguemacaroon) [reply index.html]: removed "hidden" class from each button and left it on "btn-container" - resolved this issue.

File: index.html (hunk 2 above)
Comment (roguemacaroon) [reply index.html]: removed padding from button 1 in global.css and it resolved the issue

File: css/home.css (hunk 1 above)
Comment (roguemacaroon) [reply css/home.css:788]: added "a.image-link" with a var of white and resolved the issue

File: index.html (hunk 4 above)
Comment (roguemacaroon) [reply index.html]: the navigation was merged in on line 20 (after original PR was sent), so this is done and resolved :)

**Review by maxharrisnet — APPROVED** [review]
LGTM, nice work!

**Pull request merged**